embedded-kafka/src/main/java/com/playtika/test/kafka/KafkaTopicsConfigurer.java

```diff
@@ -5,41 +5,51 @@
 import lombok.Getter;
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
 import org.testcontainers.containers.Container;
 import org.testcontainers.containers.GenericContainer;
 
 import javax.annotation.PostConstruct;
-
 import java.util.Collection;
+import java.util.Collections;
+import java.util.Map;
+import java.util.stream.Collectors;
 
 import static com.playtika.test.common.utils.ContainerUtils.executeInContainer;
 
 @Slf4j
 @RequiredArgsConstructor
 @Getter
 public class KafkaTopicsConfigurer {
+    private static final int DEFAULT_PARTITION_COUNT = 1;
 
     private final GenericContainer kafka;
     private final ZookeeperConfigurationProperties zookeeperProperties;
     private final KafkaConfigurationProperties kafkaProperties;
+    private final Map<String, Integer> partitionsByTopicsToCreate;
```

Comment:
Please move configuration to `KafkaConfigurationProperties`.
Also I would like this configuration to be done the following way, so that we can add more options for it in the future easily:
```
    protected List<TopicConfiguration> topicsConfiguration;

    @Data
    class TopicConfiguration {

        String topic;
        int partitions;
    }
```

And an example of how to use it:
```
embedded.kafka.topicsToCreate: secureTopic,topic1,topic2,topic3,camelTopic
embedded.kafka.secureTopics: secureTopic

embedded.kafka.topicsConfiguration:
  - topic: topic1
    partitions: 5

  - topic: topic2
    partitions: 3
```